lncfg/gossip.go

```diff
@@ -35,3 +38,17 @@ func (g *Gossip) Parse() error {
 
 	return nil
 }
+
+// Validate checks the Gossip configuration to ensure that the input values are
+// sane.
```

Comment:
Can you add a rationale why not less than 3 ?

Reply:
updated